EnglishFilter/sections/whitelist.txt

```diff
@@ -33,6 +33,8 @@ hinative.com#@#.banner_header
 @@||cxad.cxense.com/adserver/search$domain=gizmodo.jp
 ! adcripto.com link hidden
 adcripto.com#@#a[href^="https://goraps.com/"]
+! https://github.com/AdguardTeam/AdguardFilters/issues/103433
+@@||gizmochina.com/wp-content/plugins/cookie-law-info/public$domain=gizmochina.com

```

Comment:
better:
`@@||gizmochina.com/wp-content/plugins/cookie-law-info/public/js/cookie-law-info-public.js`

Reply:
and this rule should be in cookies_whitelist.txt